src/js/steps/letters/init.js

```diff
@@ -21,15 +23,34 @@ module.exports = function ($events) {
           .end()
         .eq(currentPage)
           .addClass('letter-active');
+
+      if (isMobile && icons) {
```

Comment:
You're calling `isMobile` as a function later on in the file. Not sure whether you're redefining a variable or just forgetting the function call

_shakes fist at JavaScript_